ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/port_channel_interfaces.schema.yml

```diff
@@ -255,6 +255,16 @@ keys:
                 - str
               min: 3
               max: 50
+            neighbor:
+              type: str
+              description: IPv4 or IPv6 address
```

Comment:
If it is a L3 Port-channel it is enough to just configure the `bfd neighbor` but in case it is a L2 Port-channel the globally configured BFD local address is needed `bfd local-address A.B.C.D`. Would it make sense to mention that here in the description?

Reply:
@bjmeuer Yes please write a suggestion for the updated description. Also please check if we cover the required model already. If not we should add it to this PR.

Reply:
I made a suggestion for the description. 

The model for the BFD local-address is not available today and should be added to this PR.

Reply:
Fixed